Makefile

```diff
@@ -41,6 +41,20 @@ skopeo: cmd/skopeo
 binary-local:
 	go build -ldflags "-X main.gitCommit=${GIT_COMMIT}" -o skopeo ./cmd/skopeo
 
+#   make debug
+#     Note: Uses the -N -l go compiler options to disable compiler optimizations
+#           and inlining. Using these build options allows you to subsequently
+#           use source debugging tools like delve.
+debug: debug-skopeo docs
```

Comment:
make install currently requires the doc be built first .. otherwise yeah sure we don't have to build docs